l10n/exceptions/firefox_android.json

```diff
@@ -29,7 +29,11 @@
     "strings": []
   },
   "ellipsis": {
-    "locales": {},
+    "locales": {
+      "ca": [
+      "mozilla-mobile/fenix/app/src/main/res/values/strings.xml:erase_action_cfr_message"
```

Comment:
You should fix the translation (just did), not add an exception.

Reply:
Thanks, didn't realize the ellipsis were dots - and thought the error was that he wasn't using the dash

Reply:
Updated